components/Image/image.css

```diff
@@ -0,0 +1,17 @@
+.round-img, .square-img {
+    width: 150px;
+    height: 150px;
```

Comment:
why px and not rem here?

Reply:
didn't think of it just went with pixels..but I will change it to rem that way it will respond more to window changes. @kirti2602